kv-components/vue/KvSelect.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<div>
+		<span class="relative">
```

Comment:
Small thing, but I think you can remove this span by putting the "relative" on the div above

Reply:
+1

Reply:
Wasn't able to remove the double container, but reworked it as we discussed @ryan-ludwig

Reply:
@emuvente An issue came up with conflicting classes for positioning. If you were to use KvSelect somewhere with
`<kv-select class="absolute top-0 right-0">` 
that absolute class would create a conflict if KvSelect looked like
```
<template>
  <div class="relative...">
    <select...
 ```

Keeping the outer div fairly class-free seems like something we should be mindful of for these Kv* components.